### PR TITLE
Add campaign tracking parameters for all links within Matomo app that link to matomo.org

### PR DESCRIPTION
**Pull request by bx80**
### Description:

This PR is a companion change to https://github.com/matomo-org/matomo/pull/21375

It changes links to matomo.org to include campaign parameters to track the location in the application from which the link was followed.

It depends on changes in the parent PR and should only be merged along with it.

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: vue/src/TestResultsPage/TestResultsPage.vue [hunk 1]
```diff
@@ -12,7 +12,7 @@
       <p>{{ translate('SecurityInfo_PluginDescription') }}</p>
       <p>
         Learn more: read our guide
-        <a rel='noreferrer' target='_blank' href='https://matomo.org/docs/security/'>
+        <a rel='noreferrer' target='_blank' href='{{ externalRawLink("https://matomo.org/docs/security/") }}'>
```
Comment (AltamashShaikh) [inline vue/src/TestResultsPage/TestResultsPage.vue]: @bx80 this should be :href and need more changes, I tried fixing quickly but would require more work

**Review by AltamashShaikh — CHANGES_REQUESTED** [review]
Left comment to fix

File: vue/src/TestResultsPage/TestResultsPage.vue (hunk 1 above)
Comment (bx80) [reply vue/src/TestResultsPage/TestResultsPage.vue]: Thanks for spotting this :+1: The same issue exists in a few other places. I've pushed a commit to this PR which should sort this.

**Review by AltamashShaikh — CHANGES_REQUESTED** [review]
@bx80 The vue code is still not appending any parameters
Screenshot from 2023-11-08 13-28-08

**Comment by bx80** [comment]
It's displaying the campaign parameters for me, here's a screenshot:

image

Could there be something cached?

**Review by AltamashShaikh — APPROVED** [review]
@bx80 It works for me now and I asked Jacob to check it and it works for him too

**Pull request merged**